src/Illuminate/Validation/Rules/Email.php

```diff
@@ -98,12 +99,12 @@ public static function default()
      * @param  bool  $strict
      * @return $this
```

Comment:
As well as this:
```suggestion
     * @param  bool  $strict
     * @param  bool||\Closure(): bool  $condition
     * @return $this
```
as well as for the other methods